app/src/main/java/org/simple/clinic/sync/indicator/dialog/SyncIndicatorFailureDialog.kt

```diff
@@ -0,0 +1,35 @@
+package org.simple.clinic.sync.indicator.dialog
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatDialogFragment
+import androidx.fragment.app.FragmentManager
+import org.simple.clinic.R
+
+class SyncIndicatorFailureDialog : AppCompatDialogFragment() {
+
+  companion object {
+    private const val KEY_MESSAGE = "failure_message"
+
+    fun show(fragmentManager: FragmentManager, message: String) {
```

Comment:
Will `message` be used in a future commit?

Reply:
Oh no. Good catch.
I have to pass this message :P